nativelib/src/main/resources/dirent.c

```diff
@@ -37,3 +37,14 @@ int scalanative_readdir(DIR *dirp, struct scalanative_dirent *buf) {
 }
 
 int scalanative_closedir(DIR *dirp) { return closedir(dirp); }
+
+int scalanative_gettempdir(char *buffer, size_t length)
+{
+#ifndef _WIN32
+  const char* tmpdir = "/tmp";
+  strncpy(buffer, tmpdir, length);
+  return 0;
+#else
+  return getWinTempDir(buffer, length);
```

Comment:
How about we use the `java.io.tmpdir` system property and make this code part of the platform specific stuff that it already does?

From https://docs.oracle.com/javase/7/docs/api/java/io/File.html#createTempFile(java.lang.String,%20java.lang.String,%20java.io.File)

> If the directory argument is null then the system-dependent default temporary-file directory will be used. The default temporary-file directory is specified by the system property java.io.tmpdir. On UNIX systems the default value of this property is typically "/tmp" or "/var/tmp"; on Microsoft Windows systems it is typically "C:\\WINNT\\TEMP". A different value may be given to this system property when the Java virtual machine is invoked, but programmatic changes to this property are not guaranteed to have any effect upon the temporary directory used by this method.